src/primitives/transaction.h

```diff
@@ -124,6 +124,11 @@ class CTxIn
         return !(a == b);
     }
 
+    friend bool operator<(const CTxIn& a, const CTxIn& b)
+    {
+        return a.prevout < b.prevout;
+    }
+
```

Comment:
It looks like this isn't needed at all... But I would like to avoid invalidating ACKs, so I'll leave it in unless I have to rebase or make other changes.

Reply:
Had to rebase, removed